server/routes/dischargeAddress.js

```diff
@@ -16,16 +16,16 @@ module.exports = function({logger, dischargeAddressService, licenceService, auth
         logger.debug('GET /dischargeAddress');
 
         const nomisId = req.params.nomisId;
-        const licence = await licenceService.getLicence(nomisId);
+        const rawLicence = await licenceService.getLicence(nomisId);
+        const licence = rawLicence && rawLicence.licence || null;
```

Comment:
I'm a big fan of using http://ramdajs.com/docs/#path
It would simply this to 
`const licence = path(['licence'], rawLicence)`;

You could also just import only that module.. navigating dynamic objects in a safe way it a win IMO

Reply:
Agreed